src/index.html

```diff
@@ -1,13 +1,621 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Title</title>
+    <title>NAMU</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="icon" href="./images/favicon.png">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Playfair+Display:wght@400;700&family=Raleway:ital,wght@0,400;1,600&display=swap" rel="stylesheet">
   </head>
-  <body>
-    <h1>Hello Mate Academy</h1>
-    <script type="text/javascript" src="scripts/main.js"></script>
+  <body class="page__body">
+    <script
+      type="text/javascript"
+      src="scripts/main.js"
+      defer>
+    </script>
+
+    <header class="header">
+      <div class="header__container">
+        <div class="top-actions">
+          <a href="#" class="logo">
+            <img
+              src="images/NAMU.svg"
+              alt="NAMU"
+            >
+          </a>
+
+          <div class="header__part">
+            <a href="#menu" class="icon icon--menu"></a>
+
+            <a href="#" class="header__language">UA</a>
+          </div>
+        </div>
+      </div>
+
+      <div class="header__content">
+        <div class="holder">
+          <div class="header__frame grid grid--tablet">
+            <div class="header__date grid__item grid__item--1-5">10 серпня - 10 листопада</div>
+
+            <h1 class="header__title grid__item grid__item--1-5">
+              Мистецтво ХІХ - ХХ ст.
+            </h1>
+
+            <div class="header__subtitle grid__item grid__item--1-3">
+              Внесок українських митців у
+              <br class="header__br">
+              світову
+              культуру 19-20 ст.
+            </div>
+
+            <form
+              method="get"
+              action="#visit"
+              class="grid__item grid__item--1-5"
+            >
+              <button
+                type="submit"
+                class="header__tickets"
+              >
+                Купити квиток
+              </button>
+            </form>
```

Comment:
Why do you need this form? Because of that, when users press this link, page will be reload and also unnecessary query parametrs will be added

Replace it with link